spec/src/main/asciidoc/repository.asciidoc

```diff
@@ -611,21 +608,22 @@ Here is an example where an application uses `@Query` to provide a partial query
 @Repository
 public interface CustomerRepository extends BasicRepository<Customer, Long> {
     @Query("WHERE totalSpent / totalPurchases > ?1")
-    CursoredPage<Customer> withAveragePurchaseAbove(float minimum, PageRequest<Customer> pageRequest);
+    CursoredPage<Customer> withAveragePurchaseAbove(float minimum,
+                                                    PageRequest pageRequest,
+                                                    Order<Customer> sorts);
 }
 ----
 
 Example traversal of pages:
 
 [source,java]
 ----
-PageRequest<Customer> pageRequest =
-        Order.by(_Customer.yearBorn.desc(),
-                 _Customer.name.asc(),
-                 _Customer.id.asc())
-             .pageSize(25);
+Order<Customer> order = Order.by(_Customer.yearBorn.desc(),
+                                 _Customer.name.asc(),
+                                 _Customer.id.asc());
```

Comment:
This is like, super-nitpicky and peevey, but at least to my eyes, this is much prettier to look at:
```suggestion
Order<Customer> order = 
        Order.by(_Customer.yearBorn.desc(),
                 _Customer.name.asc(),
                 _Customer.id.asc());
```
But it's up to you.